src/app/static/src/scss/publish-reports.scss

```diff
@@ -1,5 +1,5 @@
 @import "style";
-@import "partials/changelogs";
+@import "partials/_changelogs";
```

Comment:
`sass` only resolves partials that start with `_`,  hence all these name changes